package.json

```diff
@@ -1,37 +1,44 @@
 {
-    "dependencies": {
-        "prettier": "^2.7.1"
-    },
-    "name": "js",
-    "version": "1.0.0",
-    "main": "index.js",
-    "scripts": {
-        "hw16": "npx ts-node ./hw16/index.ts",
-        "hw17-test-mocha": "mocha --config='./hw17/.mocharc.json'",
-        "hw17-test-jest": "jest --config='./hw17/jest.config.cjs'"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/ViktoriyaKhvin/JS.git"
-    },
-    "author": "ViktoriyaKhvin",
-    "license": "ISC",
-    "bugs": {
-        "url": "https://github.com/ViktoriyaKhvin/JS/issues"
-    },
-    "homepage": "https://github.com/ViktoriyaKhvin/JS#readme",
-    "description": "",
-    "devDependencies": {
-        "@types/chai": "^4.3.3",
-        "@types/jest": "^28.1.6",
-        "@types/mocha": "^9.1.1",
-        "@types/node": "^18.7.2",
-        "chai": "^4.3.6",
-        "eslint": "^8.21.0",
-        "jest": "^28.1.3",
-        "mocha": "^10.0.0",
-        "ts-jest": "^28.0.7",
-        "ts-node": "^10.9.1",
-        "typescript": "^4.7.4"
-    }
+  "dependencies": {
+    "eslint": "^8.22.0",
+    "mocha": "^10.0.0",
+    "nyc": "^15.1.0",
+    "prettier": "^2.7.1",
+    "save-dev": "^0.0.1-security"
+  },
+  "name": "js",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "hw16": "npx ts-node ./hw16/index.ts",
+    "hw17-test-mocha": "mocha --config='./hw17/.mocharc.json'",
+    "hw17-test-jest": "jest --config='./hw17/jest.config.cjs'",
+    "hw18-test-jest": "jest --config='./hw18/jest.config.cjs'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ViktoriyaKhvin/JS.git"
+  },
+  "author": "ViktoriyaKhvin",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ViktoriyaKhvin/JS/issues"
+  },
+  "homepage": "https://github.com/ViktoriyaKhvin/JS#readme",
+  "description": "",
+  "devDependencies": {
+    "@types/chai": "^4.3.3",
+    "@types/jest": "^28.1.7",
+    "@types/mocha": "github:types/mocha",
+    "@types/node": "^18.7.9",
+    "@types/superagent": "^4.1.15",
+    "chai": "^4.3.6",
+    "jest": "^28.1.3",
+    "mochawesome": "^7.1.3",
+    "superagent": "^8.0.0",
+    "ts-jest": "^28.0.8",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.7.4"
```

Comment:
You have some unused packages: `nyc`, `mochawesome`, save-dev`. Please delete them.